js/adapt-contrib-slider.js

```diff
@@ -370,22 +370,33 @@ define([
             var $markers = this.$('.slider-markers').empty();
             if (this.model.get('_showScale') === false) {
                 $markers.eq(0).css({display: 'none'});
-                this.$('.slider-scale-numbers').eq(0).css(
+                this.$('.slider-scale-numbers *:not(".slider-scale-marker")').css(
                     this.model.get('_showScaleIndicator') ? {visibility: 'hidden'} : {display: 'none'}
                 );
-            } else {
-                var $scaler = this.$('.slider-scaler');
-                for (var i = 0, count = this.model.get('_items').length; i < count; i++) {
-                    $markers.append("<div class='slider-line component-item-color'>");
-                    $markers.find('.slider-line').eq(i).css({left: this.mapIndexToPixels(i, $scaler) + 'px'});
-                }
-                var scaleWidth = $scaler.width();
-                var $numbers = this.$('.slider-scale-number');
-                for (var j = 0, len = this.model.get('_items').length; j < len; j++) {
-                    var $number = $numbers.eq(j);
-                    var newLeft = Math.round($number.data('normalisedPosition') * scaleWidth);
-                    $number.css({left: newLeft});
-                }
+                return;
+            }
+
+            var $scaler = this.$('.slider-scaler');
+            for (var i = 1, count = this.model.get('_items').length - 1; i < count; i++) {
+                $markers.append("<div class='slider-line component-item-color' style='left: " + this.mapIndexToPixels(i, $scaler) + "px'>");
+            }
+            // Do we show scale numbers
+            this.showScaleNumbers();
+        },
+
+        showScaleNumbers: function () {
+            var $scaler = this.$('.slider-scaler');
+            if (this.model.get('_showScaleNumbers') === false) {
+                this.$('.slider-scale-numbers *:not(".slider-scale-marker")').css('display', 'none');
+                return;
+            }
+
+            var scaleWidth = $scaler.width();
+            var $numbers = this.$('.slider-scale-number');
+            for (var j = 0, len = this.model.get('_items').length; j < len; j++) {
```

Comment:
sorry didn't spot this before - but you don't need to be using `j` as your variable here any more - although you actually might as well switch over to `this.model.get('_items').forEach` for much cleaner looking code

Reply:
Thanks @moloko updated 👍